pwa/app/routes/about.tsx

```diff
@@ -0,0 +1,158 @@
+            Competition (FRC) with scouting data and match videos. Since then,
+            the project has grown to include developers from within the FIRST
+            community from various teams all over the world. We continually
+            strive to make The Blue Alliance an even more valuable resource for
+            our users. Other additions to The Blue Alliance include match
+            prediction and analytics, hosting of CAD models, and The Blue
+            Alliance Blog.
+          </p>
+          <p>
+            Since 2022, The Blue Alliance has been a 501(c)(3) non-profit
+            organization. We rely on community donations to pay for hosting
+            costs.
+          </p>
+
+          <p>
+            You can support The Blue Alliance or reach us with the following:
+          </p>
+          <a
+            href="https://github.com/the-blue-alliance"
+            className={cn(
+              buttonVariants({ variant: 'default', size: 'default' }),
+            )}
+          >
+            Contribute on GitHub!
+          </a>
+          <Link
+            to="/donate"
+            className={cn(
+              buttonVariants({ variant: 'default', size: 'default' }),
+            )}
+          >
+            Donate with PayPal
+          </Link>
+          <Link
+            to="/contact"
+            className={cn(
+              buttonVariants({ variant: 'default', size: 'default' }),
+            )}
+          >
+            Contact Us!
+          </Link>
+        </div>
+
+        <div>
+          <h3 className="text-2xl">
+            About <em>FIRST</em> and <em>FIRST</em> Robotics Competition
+          </h3>
+          <p>
+            <a href="http://www.firstinspires.org/">
+              {' '}
+              <em>FIRST</em>
+            </a>{' '}
+            (For Inspiration and Recognition of Science and Technology) is a
+            non-profit organization with the mission of inspiring young people
+            to be science and technology leaders. With programs that involve
+            students from kindergarten through high school, FIRST has become a
+            world-wide phenomenon with teams from all six inhabited continents.
+          </p>
+          <p>
+            The{' '}
+            <a href="https://www.firstinspires.org/robotics/frc">
+              <em>FIRST</em> Robotics Competition (FRC)
+            </a>{' '}
+            gives high school students and their adult mentors the opportunity
+            to work and create together to solve a common problem. Each{' '}
+            <em>FIRST</em> Robotics Competition season culminates with local and
+            regional events where qualifying teams compete for awards and a spot
+            in the <em>FIRST</em> Championship.
+          </p>
+
+          <div className="flex">
+            <img
+              src="https://www.firstinspires.org/sites/default/files/uploads/resource_library/brand/thumbnails/FIRST-V.png"
```

Comment:
I think it’s better to move this to our assets directory.

Reply:
are you referring to `/pwa/public`